src/igvfd/upgrade/donor.py

```diff
@@ -33,3 +33,17 @@ def human_donor_2_3(value, system):
     # https://igvf.atlassian.net/browse/IGVF-90
     if 'health_status_history' in value:
         del value['health_status_history']
+
+
+@upgrade_step('human_donor', '3', '4')
```

Comment:
this is supposed to be a single function, not two separate functions. do something like: ```
@upgrade_step('phenotype_term', '1', '2')
@upgrade_step('assay_term', '1', '2')
@upgrade_step('sample_term', '1', '2')
def ontology_term_1_2(value, system):
    # https://igvf.atlassian.net/browse/IGVF-221
    if 'aliases' in value:
        if len(value['aliases']) == 0:
            del value['aliases']
```

Reply:
done